notifier/src/lib.rs

```diff
@@ -92,9 +93,21 @@ enum NotificationType {
     Log(Level),
 }
 
+#[derive(Clone)]
+pub enum NotificationType {
+    Trigger(String),
+    Resolve(String),
+}
+
+impl Default for NotificationType {
```

Comment:
I'm curious why a `Default` implementation was needed?

Reply:
It's not, I was trying to implement a different way around initially where it would've been used. Will remove it